dev/com.ibm.ws.jsfContainer_fat_2.3/publish/files/MANIFEST_badMyfacesImpl_40.MF

```diff
@@ -0,0 +1,264 @@
+Manifest-Version: 1.0
+Archiver-Version: Plexus Archiver
+Created-By: Apache Maven Bundle Plugin
+Built-By: siedlecki
+Build-Jdk: 11.0.13
+Specification-Title: Apache MyFaces Core 4.0 - Impl
+Specification-Version: 3.0.0
```

Comment:
Everything is the same as the RC2 Manifest except for `Specification-Version` since the test is expecting a bad version.